lib/lib.es2018.d.ts

```diff
@@ -20,4 +20,5 @@ and limitations under the License.
 
 /// <reference path="lib.es2017.d.ts" />
```

Comment:
do not update the lib files. they are generated as part of the build (`jake LKG`) and should not be edited manually.

Reply:
that applies to all files `lib\*`

Reply:
Aha.. That makes sense. 

I had a feeling that was the case when editing the file.. 80,000+ lines! 

Would you like me to revert these?

Reply:
yes please.